formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -157,21 +157,24 @@ let acceptsClosedAnonObjAsArg
     (o: < x : int, y : int >) =>
   o#x + o#y;
 
-let res = acceptsOpenAnonObjAsArg {
-  method x = 0;
-  method y = 10
-};
+let res =
+  acceptsOpenAnonObjAsArg {
+    method x = 0;
+    method y = 10
+  };
 
-let res = acceptsOpenAnonObjAsArg {
```

Comment:
FWIW, I prefer the new version.  I still like the rule that anything that does not fit on a single line should have only whitespace to the left of its start.  If it was up to me, I would not dock anything at the end of a line except a brace, paren, or bracket.